prioritizer/prioritizer.py

```diff
@@ -120,32 +81,72 @@ def quantitative_score(city, action):
     Returns:
         float: The calculated quantitative score for the action.
     """
```

Comment:
This comment is for quantitative score:
Is there a decision made somewhere why we only use those four
1. Emissions reduction score: Based on the GHG reduction potential of the action.
 2. Adaptation effectiveness score: Based on the adaptation effectiveness of the action.
 3. Time in years score: Based on the timeline for implementation of the action.
 4. Cost score: Based on the budget of the city.
 For ranking? E.g. what about CoBenefits?

Reply:
Also it says ` Cost score: Based on the budget of the city.` but I do not see how the city data is used in this ranking?
Are we using the ccra hazards already (mapping the ccra of the city to the addressed hazards of the action?)